prijateli_tree/app/routers/games.py

```diff
@@ -407,9 +415,6 @@ def integrated_game(
     # Get current round
```

Comment:
This function / route isn't going to be used right? You're going to have them go back and forth between get round, post answer, WAIT, get round, post answer WAIT, until the end of the game.  Where we add a score @router.post("/{game_id}/player/{player_id}/update_score") redirect to  @router.get("/{game_id}/player/{player_id}/score").